components/count-badge/count-badge.js

```diff
@@ -134,7 +164,11 @@ class CountBadge extends RtlMixin(LitElement) {
 			numberString = `${'9'.repeat(this.maxDigits)}+`;
 		}
 		return html`
-        	<div class="d2l-count-badge-number">${numberString}</div>`;
+        	<div class="d2l-count-badge-number" 
+				tabindex="${this.tabStop ? '0' : '-1'}" 
```

Comment:
If you don't need to programmatically focus on the element, you could leave it `undefined` + the `ifDefined` directive to just set it to `0` when `tabStop` is `true`.

```html
tabindex="${ifDefined(this.tabStop ? '0' : undefined)}" 
```